examples/transaction.rs

```diff
@@ -21,23 +19,13 @@ async fn get(client: &Client, key: Key) -> Option<Value> {
     txn.get(key).await.expect("Could not get value")
 }
 
-// Ignore a spurious warning from rustc (https://github.com/rust-lang/rust/issues/60566).
-#[allow(unused_mut)]
-async fn scan(client: &Client, range: impl RangeBounds<Key>, mut limit: usize) {
+async fn scan(client: &Client, range: impl Into<BoundRange>, mut _limit: usize) {
     let mut txn = client.begin().await.expect("Could not begin a transaction");
     txn.scan(range)
-        .into_stream()
-        .take_while(move |r| {
-            assert!(r.is_ok(), "Could not scan keys");
-            future::ready(if limit == 0 {
-                false
-            } else {
-                limit -= 1;
-                true
-            })
-        })
-        .for_each(|pair| future::ready(println!("{:?}", pair)))
-        .await;
+        .await
+        .unwrap()
```

Comment:
Could you use `expect` instead of `unwrap` please?

Reply:
Fixed 35eeceb